test/fixtures/basic/unmanaged-pod.yml.erb

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: unmanaged-pod-<%= deployment_id %>
+  labels:
+    type: unmanaged-pod
+    name: unmanaged-pod-<%= deployment_id %>
+    app: basic
+spec:
+  activeDeadlineSeconds: 20
+  restartPolicy: Never
+  containers:
+    - name: basic
+      image: hello-world:latest
```

Comment:
This is how we do migrations for all our apps using Cloudsql: launch a pod that runs `rake db:migrate` and exits, wait for it to finish, and dump its logs inline.